### PR TITLE
Add exportIdentity() to ConnectionManager and AdminConnection

### PR DESCRIPTION
**Pull request by bestbeforetoday**
This function is currently implemented only by the Hyperledger Fabric v1 connection manager.

Required for #1427.

Signed-off-by: Mark S. Lewis <Mark_Lewis@uk.ibm.com>

### CODE REVIEW

**Comment by bestbeforetoday** [comment]
@davidkel Simon specifically asked me to rename it to _publicKey_ to be consistent with the rest of the current API. The idea being we change the naming everywhere together later.

It wasn't obvious to me which other connectors needed exportIdentity() added since all of them seem to be missing some number of the abstract functions from the base class. What is the definitive list?

**Comment by sstone1** [comment]
@bestbeforetoday I forgot about the proxy connector - the following connectors have `importIdentity` implementations and should have `exportIdentity` implementations:

`composer-connector-embedded`
`composer-connector-hlfv1`
`composer-connector-proxy`
`composer-connector-web`

I am in the process of removing `composer-connector-hlf` so don't bother updating that one.

File: packages/composer-systests/systest/identities.js [hunk 1]
```diff
@@ -273,4 +273,45 @@ describe('Identity system tests', () => {
...
+        let profileName;
+        let certificate;
+        let privateKey;
+        if (TestUtil.isHyperledgerFabricV1()) {
+            profileName = 'composer-systests-org1';
+            const certificateFile = path.resolve(__dirname, '../hlfv1/crypto-config/peerOrganizations/org1.example.com/users/User1@org1.example.com/msp/signcerts/User1@org1.example.com-cert.pem');
+            certificate = fs.readFileSync(certificateFile, 'utf8').replace(/\r/g, '');
+            const privateKeyFile = path.resolve(__dirname, '../hlfv1/crypto-config/peerOrganizations/org1.example.com/users/User1@org1.example.com/msp/keystore/key.pem');
+            privateKey = fs.readFileSync(privateKeyFile, 'utf8').replace(/\r/g, '');
+        } else if (TestUtil.isWeb()) {
```
Comment (sstone1) [inline packages/composer-systests/systest/identities.js]: I think this `else if` can be removed now that you've added `exportIdentity` support for the embedded, web, and proxy runtimes? You need to manually merge the `api.txt` changes so worth a try to see if it works 😉

**Comment by sstone1** [comment]
@bestbeforetoday the embedded/web/proxy system test just seems to hang and then get timed out by Travis - can you take a look please?

**Pull request merged**